doozer/doozerlib/backend/konflux_client.py

```diff
@@ -401,6 +407,7 @@ async def start_pipeline_run_for_image_build(
         git_auth_secret: str = "pipelines-as-code-secret",
         additional_tags: Sequence[str] = [],
         skip_checks: bool = False,
+        pipelinerun_template_url: str = constants.KONFLUX_DEFAULT_IMAGE_BUILD_PLR_TEMPLATE_URL,
```

Comment:
nit: Might need to update docstring as well, to add in `pipelinerun_template_url`